Tutorial_1b_Moment_calculation.ipynb

```diff
@@ -0,0 +1,860 @@
+{
```

Comment:
<p><code> follows a Gaussian distribution as well: </code> This could be cut, IMO, as the line shape and distribution aren't really comparable and may lead to confusion.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>